fili-core/src/main/java/com/yahoo/bard/webservice/table/BaseSchema.java

```diff
@@ -23,6 +24,24 @@ protected BaseSchema(Set<Column> columns) {
     }
 
     @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof BaseSchema)) {
+            return false;
+        }
+
+        BaseSchema that = (BaseSchema) o;
+        return Objects.equals(columns, that.columns);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(columns);
+    }
```

Comment:
Comparing schemas during test causes `Object` comparison which is not we want, in addition to this one, we need to override `equals` and `hashCode` in all subclass schemas which is not done yet and needs to do.